CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.7.0 (August 27, 2020) - Fixes
```

Comment:
Let's do like that:

```
## [Unreleased]

### Fixes
```

Reply:
sorry for not mentioning, but we decided that's it's better to have release branches, so we just add what you fixed under ### Fixes and we add the version later

Reply:
ok